roles/vmware-ci-configure-ansible-test/templates/cloud-config-vcenter.ini.j2

```diff
@@ -0,0 +1,15 @@
+[DEFAULT]
+vcenter_username: administrator@vsphere.local
+vcenter_password: {{ lookup('file', vmware_ci_set_passwords_secret_dir + '/vcenter/tmp/vcenter_password.txt') }}
```

Comment:
why does this have to stored on disk and use lookup? This could leak info (which isn't big deal, more curious)

Reply:
We are dealing with several several Ansible processes, on different hosts. Sharing variables is a giant pain in the neck. I came to the conclusion that this is best approach.